BusinessAdapter.FP.UnitTests/Parsing/FpFileParserTests.Fixture.cs

```diff
@@ -54,8 +54,9 @@ public string CreateExampleCIMAcknowledgeMessage()
         
         public string CreateExampleCIMScheduleMessage()

```

Comment:
Die Methode erstellt die Datei nicht wirklich sondern baut nur den Pfad zusammen.
Typischerweise wird das in einer TestData-Klasse gekapselt.
Siehe: https://developer-campus.de/tracks/programming/services/implementierungsmuster/testing/teststruktur-unittests/